apps/earn-protocol/components/layout/BridgePageView/BridgePageViewLoadingState.tsx

```
@@ -0,0 +1,57 @@
+'use client'
+
+import { type FC } from 'react'
+import { Button, Card, SkeletonLine } from '@summerfi/app-earn-ui'
+
+import BridgeFormTitle from '@/features/bridge/components/BridgeFormTitle/BridgeFormTitle'
+import { Spacer } from '@/features/bridge/components/Spacer/Spacer'
+
+import classNames from './BridgePageView.module.scss'
+
+interface BridgePageViewLoadingStateProps {}
```

Biome (1.9.4)

[error] An empty interface is equivalent to {}.

Safe fix: Use a type alias instead.


(lint/suspicious/noEmptyInterface)